k8s/api-deployment.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    io.kompose.service: api
+  name: api
+spec:
+  replicas: 1
+  selector:
+    matchLabels: 
+      io.kompose.service: api
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        io.kompose.service: api
+    spec:
+      containers:
+      - args:
+        - uwsgi
+        - --uid
+        - guest
+        - --gid
+        - nogroup
+        - --http
+        - :9876
+        - --socket
+        - :9877
+        - --manage-script-name
+        - --mount
+        - /app=monocle.webapp:app
+        image: changemetrics/monocle_backend:latest
```

Comment:
Images are published automatically for each stable release (tag) but also for each master. https://github.com/change-metrics/monocle/actions/workflows/publish-master.yaml
Images are published here: https://hub.docker.com/u/changemetrics
I think this k8s manifest should use the "latest" container tag for both images and pull from dockerhub.

Reply:
can I resolve this? @morucci @georgettica

Reply:
We're you able to pull the image suggested? Or did you need to modify it to be image pullable..

If this needs an image change, do.the change

Reply:
@morucci since docker has this Pull Limit rate, we are unable to pull the image. 
To workaround this we can either: sync it to Quay, or to download it into the openshift internal registry, or change pull method to be less aggressive.